src/create_disk.sh

```diff
@@ -544,16 +544,58 @@ create_dmverity() {
     mount -o ro "/dev/mapper/${partlabel}" "${mountpoint}"
 }
 
+# Save genprotimg input for later and don't run zipl here
+rdcore_replacement() {
+    local se_kargs_append=("ignition.firstboot")
+    while [ $# -gt 0 ]; do
+        se_kargs_append+=("$1")
+        shift
+    done
+
+    local se_tmp_disk se_tmp_mount blsfile
```

Comment:
For consistency, let's put the `local` declarations on the lines that set the variable values.

Reply:
I did that, however shellcheck does not like it, since it would mask the rc value of the commands.
I could instead put the other variables into the declaration line as well for consistency.

Reply:
Ah, okay.  Yes, let's be consistent the other way then.